prow/cluster/hook_deployment.yaml

```diff
@@ -33,7 +33,7 @@ spec:
       terminationGracePeriodSeconds: 180
       containers:
       - name: hook
-        image: gcr.io/k8s-prow/hook:v20180302-0064877cc
+        image: gcr.io/k8s-prow/hook:v20180302-5519b614a
```

Comment:
why did this change? 0064877cc -> 0064877cc7181a16a6d753d111eb1381bfdaa148, which is already past the milestone updates.